### DISCUSSION TITLE
rules validation not work after compilation

### DISCUSSION THREAD

**Discussion by majweb** [react-hook-form/react-hook-form#6564] [Q&A]
I warmly welcome
I have a problem with react form hook v5 ...
I have a controlled component and an initial state with value'

false  (extraCityShow)
<Controller
    as={<Select options={country} />}
    control={control}
   rules={{
               required:
               {
               value: extraCityShow == false,
               message: 'Wybierz kraj'
               }
               }}

I work locally and it works great. After compiling, minifying and uploading to VPS it doesn't work. Has anyone encountered a similar situation?

**Comment by bluebill1049** [comment]
can you reproduce this in a repo or something we could easily reproduce?
